### ISSUE TITLE
Some rates are messed up

### ISSUE THREAD

**Issue by danielkucera** [azonenberg/scopehal#536]
When I configure rates 1,25Msps, 2,5Msps, 12,5Msps they result in command RATE 1, RATE 3, RATE 13.
Is this some feature or a bug?

See attached video

https://user-images.githubusercontent.com/1734256/150169049-9e715a14-2f13-4de9-b9a9-188212d09ce9.mp4

.

**Comment by azonenberg** [comment]
This is in the Pico driver I assume, since you didn't specify?

**Comment by danielkucera** [comment]
Yes, I am using pico driver but I couldn't find it there. Some substitutions or such...
